New blog post: 2022 and Kubewarden (#166)
kubewarden/kubewarden.io: content/blog/2022/12/kubewarden-during-2022.md

---

> title: Another year of Kubewarden

Author: Good point, I've changed that

authors:
- Flavio Castelli
date: 2022-12-28
---

With 2022 being basically over, it's time to look back at what happened within
the Kubewarden project during the last twelve months.

> ## The 1.0 release, CNCF inclusion

Author: Done

I think the major achievements of the project have been reaching the maturity
state, which happened with its 1.0 release during the month of July, and getting
[accepted inside of the CNCF](/blog/2022/06/cncf-sandbox-inclusion/).

The CNCF inclusion lead to an exciting integration with
[ArtifactHub](/blog/2022/07/artifact-hub-supports-kubewarden/).
This proved to be a great improvement not just for policy authors, but also
for the all Kubewarden's users who can now find all the existing policies
on the [familiar ArtifactHub interface](https://artifacthub.io/packages/search?kind=13&sort=relevance&page=1).

> By being part of CNCF, Kubewarden got also access to another CNCF project:

Author: I think there was some hiccup with GH: this suggestion was repeated a couple of times. I've accepted the 1st one

> By being part of CNCF, Kubewarden got also access to another CNCF project:

Author: I think there was some hiccup with GH: this suggestion was repeated a couple of times. I've accepted the 1st one

[CLOMonitor](https://clomonitor.io/projects/cncf/kubewarden).

> Thanks to CLOMonitor, Kubewarden's users can have a clear picture of the health

Reviewer: ```suggestion
Thanks to CLOMonitor, Kubewarden's users have a clear picture of the project health
```

> Thanks to CLOMonitor, Kubewarden's users can have a clear picture of the health

Reviewer: ```suggestion
Thanks to CLOMonitor, Kubewarden's users have a clear picture of the project health
```

> of the project and assess its development and security practices.

Reviewer: ```suggestion
and can assess its development and security practices.
```

> of the project and assess its development and security practices.

Reviewer: ```suggestion
and can assess its development and security practices.
```

Author: I think there was some hiccup with GH: this suggestion was repeated a couple of times. I've accepted the 1st one

> of the project and assess its development and security practices.

Reviewer: ```suggestion
and can assess its development and security practices.
```

Author: I think there was some hiccup with GH: this suggestion was repeated a couple of times. I've accepted the 1st one

## Pod Security Policy replacement

During the month of August, Kubernetes 1.25 release finally removed
[Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/).

> This event was planned and advertised since a long time. Due to that,

Reviewer: ```suggestion
This event was planned and advertised for a long time. Due to this,
```

the Kubewarden developers were able to provide

> [tooling, policies and documentation](https://docs.kubewarden.io/tasksDir/psp-migration)

Reviewer: ```suggestion
[tooling, policies, and documentation](https://docs.kubewarden.io/tasksDir/psp-migration)
```

> to smooth the migration process from Kubernetes Pod Security Policies to

Reviewer: ```suggestion
to smoothen the migration process from Kubernetes Pod Security Policies to
```

Kubewarden policies.

## Secure supply chain

The topic of secure supply chain started to emerge near the end of 2021, but it
was during this year that many projects and organizations tackled it.

The [sigstore](https://sigstore.dev) project proved to be one of the pillars of
these initiatives.

> Kubewarden invested early on Sigstore, with some of our maintainers being

Reviewer: ```suggestion
Kubewarden invested early in the Sigstore project, with some of our maintainers being
```
